src/App.js

```diff
@@ -1,22 +1,54 @@
-import { FormSetDates } from "./components/Form/FormSetDates";
+import {FormInput} from './components/activity/FormInput';
+import {ActivityList} from './components/activity/ActivityList';
+import {useState} from 'react';
+import styled from 'styled-components';
+import {ThemeProvider} from 'styled-components';
+import {StyleHeader} from './components/activity/styles/Header.styles';
 
+const theme = {
+  colors: {
+    header: 'white',
+    body: 'rose',
+    footer: 'lightgrey',
+  },
+};
 
+export default function App() {
+  const [activities, setActivity] = useState([]);
 
+  const addActivity = activity => {
+    setActivity([...activities, activity]);
+  };
+  const removeActivity = activity => {
+    console.log(activity);
+    setActivity(current => current.filter(obj => obj.id !== activity));
+  };
 
-export default function App() {
   return (
-    <>
-    <heading>
-      <h1>Struct your Day</h1>
-    </heading>
-    
-    
-    
+    <StyledWrapper>
+      <ThemeProvider theme={theme}>
+        <StyleHeader>
+          <img width={200} height={200} src="./images/logo.png" alt="logo" />
+          <h1> Struct your Day</h1>
```

Comment:
I'm not sure if it was intended this way, but I think it would be more correct:
```suggestion
          <h1>Structure your Day</h1>
```